### PR TITLE
fix(test): Set meaningful density

### PR DESCRIPTION
**Pull request by lnist**

### CODE REVIEW

File: test/lib/MangroveTest.sol [hunk 1]
```diff
@@ -79,7 +79,7 @@ contract MangroveTest is Test2, HasMgvEvents {
     defaultTickscale: 1,
     gasprice: 40,
     gasbase: 50_000,
-    density: 10,
+    density: 2 ** 32,
```
Comment (espendk) [inline test/lib/MangroveTest.sol:82]: I'm surprised this doesn't affect more tests!

Comment (espendk) [reply test/lib/MangroveTest.sol:82]: Why/how was this density chosen?

**Review by espendk — COMMENTED** [review]
The implications of the density change in `MangroveTest` is unclear to me. Can you say a little about how the density was chosen and how this will impact tests going forward?

File: test/lib/MangroveTest.sol (hunk 1 above)
Comment (lnist) [reply test/lib/MangroveTest.sol:82]: At random, bigger than what it was :)

Comment (lnist) [reply test/lib/MangroveTest.sol:82]: The tokens have fictional values, so the importance is that asserts can be made regarding density, not the exact value. Previously it was effectively 0.

Comment (espendk) [reply test/lib/MangroveTest.sol:82]: Ok :-) We can adjust later if needed

**Review by espendk — APPROVED** [review]
lgtm

**Pull request merged**